custom_model_runner/datarobot_drum/runtime_parameters/runtime_parameters.py

```diff
@@ -42,3 +50,73 @@ def get(key):
             )
 
         return transformed_env_value["payload"]
+
+    @classmethod
+    def mangled_param_name(cls, param_name):
+        return f"{cls.PARAM_PREFIX}{param_name}"
+
+
+class RuntimeParametersLoader:
+    """
+    This class is used by DRUM to load runtime parameter values from a provided YAML file. It is
+    used by DRUM only when executing externally to DataRobot, in a local development environment.
+
+    Here is an example for such `runtime_param_values.yaml` file:
+    ```
+    PARAM_STR_1: "Example for a string value"
+    PARAM_CRED_1:
+      credentialType: s3
+      awsAccessKeyId: ABDEFGHIJK...
+      awsSecretAccessKey: asdjDFSDJafslkjsdDLKGDSDlkjlkj...
+      awsSessionToken: null
+    """
+
+    def __init__(self, values_filepath):
+        if not values_filepath:
+            print("Empty runtime parameter values file path!")
+            exit(-1)
```

Comment:
why is this doing print and exit instead of raising an error?

Reply:
I agree, I don't really love classes/libraries calling `exit()`. Why not raise an exception and catch them at the client of this class, and call the exit there.

Also another nit is why the `-1` exit status? I always find those confusing because they just overflow when you check the status in the shell, i.e. `-1 == 255, -2 == 254, etc`.  Why not just use positive numbers to avoid the confusion?